classes/iron-flex-layout.html

```diff
@@ -8,6 +8,10 @@
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
 
+<script>
+  console.warn('This file is deprecated, and will be deleted on October 1st, 2015. Please use `iron-flex-layout/classes/iron-flex-layout-classes.html`, and one of the specific dom-modules instead');
```

Comment:
This PR isn't looking like it's merging any time soon, but I agree -- I'll add more time.